code/main.py

```diff
@@ -78,6 +78,9 @@ def __init__(self, nbr_fish, nbr_predators, boundaries, ann_weights, graphics_on
             fish = Fish(pos, velocity, i, self, ann, fish_image, self.sprite_batch_fishes)
             self.fish_lst.append(fish)
 
+        # Initialize physical constants of  Coloumb force equation
+        self.k = 1
```

Comment:
No. It's part of the environment. Makes more sense that physics is part of the environment right?